frontend/src/components/Dashboard.vue

```diff
@@ -1,24 +1,49 @@
 <template>
-  <template
-    v-if="loading"
-  >
+  <template v-if="loading">
     <Spinner />
   </template>
 
-  <div
-    v-if="!loading"
-    id="content-container"
-  >
-    <template
-      v-for="content in contents"
-      :key="content"
+  <template v-if="!loading">
+    <div class="content-container">
+      <template
+        v-for="content in visibleContent"
```

Comment:
Please use plural from for list, ex: **visibleContents**

Reply:
Done